compiler/modules/types/listProj.bal

```diff
@@ -27,8 +27,41 @@ function listProjBdd(Context cx, IntSubtype|true k, Bdd b, Conjunction? pos, Con
     }
 }
 
+final ListMemberTypes LIST_MEMBER_TYPES_ALL_TOP = [[{ min: 0, max: int:MAX_VALUE }], [TOP]];
+final ListMemberTypes LIST_MEMBER_TYPES_ALL_NEVER = [[{ min: 0, max: int:MAX_VALUE }], [NEVER]];
+
+public function listAllMemberTypes(Context cx, SemType t) returns ListMemberTypes {
```

Comment:
What is this doing in listProj.bal? Please put it back where it was in core.bal.

Reply:
Isn't this a counterpart of `listProj` in `listProj.bal:4`?